### ISSUE TITLE
lobos.utils/when->> collides with clojure 1.5.0-beta1's when->> macro

### ISSUE THREAD

**Issue by tobias** [budu/lobos#53]
This prevents lobos from being loaded under 1.5.0. I looked at not defining when->> under 1.5.0, but it unfortunately has different semantics than the clojure when->>.

**Comment by tobias** [comment]
This may be a non-issue - it looks like when->> may become some->> (https://groups.google.com/d/msg/clojure/67JQ7xSUOM4/7v3M96EfJq8J). Let's wait and see.

**Comment by budu** [comment]
Yep, Rich has renamed it two days ago, I'll be waiting for the next beta before adding a lein2 profile for 1.5. I'm leaving this issue open as a remainder. Thanks for pointing this out.
